drivers/ovsdriver.go

```diff
@@ -40,11 +40,19 @@ const (
 	hostPortName = "contivh0"
 )
 
+//EpInfo contains the ovsport and id of the group
+type EpInfo struct {
+	Ovsportname string
```

Comment:
Please add `json:"..."` attributes to the filed to be consistent with rest of the file..